### ISSUE TITLE
Release 1.0.0

### ISSUE THREAD

**Issue by ryanccarelli** [ryanccarelli/ssljax#1]
Pytorch libraries:
1. VISSL https://github.com/facebookresearch/vissl/tree/master/vissl
2. Pytorch Lightning https://lightning-bolts.readthedocs.io/en/latest/self_supervised_models.html

API
- [ ] ssljax.models.projection
  - [ ] MLP + Softmax
- [ ] ssljax.models.backbone
  - [ ] ViT
  - [ ] Sparse ViT
  - [ ] Vision Longformer
  - [ ] Swin Transformer
  - [ ] ResNext
  - [x] #2
- [ ] ssljax.augment [just wrap vissl.data?]
- [ ] ssljax.optimizers
- [ ] ssljax.trainers
  - [ ] Student-Teacher Schemes
    - [ ] DINO
    - [ ] BYOL

Repository
- [ ] pytest (sphinx napolean google style https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)
- [ ] docs
- [ ] hooks
- [ ] contributing
- [ ] license
- [ ] CI
- [ ] pypi

Models
- [ ] BYOL https://arxiv.org/abs/2006.07733
- [ ] SimCLR https://arxiv.org/abs/2002.05709
- [ ] DINO https://arxiv.org/abs/2104.14294
- [ ] ESVIT https://arxiv.org/abs/2106.09785
- [ ] SEER https://arxiv.org/abs/2103.01988
- [ ] MoCo https://arxiv.org/abs/2003.04297
- [ ] SwAV https://arxiv.org/abs/2006.09882

Examples
- [ ] Example notebooks for some basic models
